index.html

```diff
@@ -37,31 +44,40 @@ <h1 id="title"> freecode survay </h1>
 
 <p> what describes your choice option </p>
     <div class= "first"> 
+
       <select id= "dropdown" size="4"> 
+
 <option> first choice </option>
 <option> second choice </option>
 <option> third choice </option>
 <option> forth choice </option>
-<option> fifth choice </option>
-<option> sixth choice </option>
+
+<!-- <option> fifth choice </option> -->
+<!-- <option> sixth choice </option> -->
+
       </select>
     </div>
 
     <p> what is your gender </p>
     <label for ="gender"> Male <input value="gender" type="radio" name="name"> <br>
 
     <label for ="gender"> Female <input value="gender" type="radio" name="name"> <br>
+
     <p> What would you like to see improved? (Check all that apply) </p>
+
     <label for ="gender"> Option one <input value="one" type="checkbox" name="name"> <br>
     <label for ="gender"> Option two <input value="two" type="checkbox" name="name"> <br>
     <label for ="gender"> Option three <input value="three" type="checkbox" name="name"> <br>
     <label for ="gender"> Option four <input value="one" type="checkbox" name="name"> <br>
     <label for ="gender"> Option five <input value="one" type="checkbox" name="name"> <br>
     <label for ="gender"> Option six <input value="one" type="checkbox" name="name"> <br>
+
     <textarea id= "survay-form" cols="40" rows = "5"> comment here</textarea>
     <button id= "submit" type="submit"> sumbmit </button> 
 
 
+    
+
        </form>
   </body>
   <P>Thanks for help</P>
```

Comment:
kindly ensure you use proper HTML semantics, the <p>  tag is supposed to be within the HTML body